scripts/myMangaDex.js

```diff
@@ -163,9 +164,9 @@ class MyMangaDex {
 				}
 
 				let isNext = (realChapter < 0 ||
-					realChapter == this.manga.lastMyAnimeListChapter ||
-					(realChapter == this.manga.lastMyAnimeListChapter + 1 ||
-						(!this.options.saveOnlyHigher && realChapter == this.manga.lastMyAnimeListChapter - 1)));
+					//realChapter == this.manga.lastMyAnimeListChapter ||
```

Comment:
```suggestion
				realChapter == this.manga.lastMyAnimeListChapter ||
					(realChapter == this.manga.lastMyAnimeListChapter + 1 ||
						(!this.options.saveOnlyHigher && realChapter == this.manga.lastMyAnimeListChapter - 1)));
```
I don't think these belong but I thought I'd tell you I removed them, maybe there is a reason they're here

Reply:
This was used to check if it was the next chapter, iirc, but this code is a mess :)